src/hearings/containers/request-hearing/location-search-container/location-search-container.component.ts

```diff
@@ -1,16 +1,29 @@
 import { Component, OnInit } from '@angular/core';
-import { select, Store } from '@ngrx/store';
+import { FormBuilder, FormGroup } from '@angular/forms';
+import { Store, select } from '@ngrx/store';
+import { LocationModel } from 'api/locations/models/location.model';
```

Comment:
```suggestion
import { LocationByEPIMSModel } from '@hmcts/rpx-xui-common-lib/lib/models/location.model';
```

Reply:
This is not resolved yet, we don't need LocationModel  anymore in angular, and it should refer to @hmcts/rpx-xui-common-lib/lib/models/location.model not api/locations/models/location.model

Reply:
Have you completed looking at this PR @rayliangatos. So I do not want to go backwards and forwards?

Reply:
I suppose I haven't asked you backwards changes, but if you don't want repeat comments please resovle the comments at a go.